components/certifier/actions/issue-credential/issue-credential.mjs

```diff
@@ -0,0 +1,134 @@
+import { defineAction } from "@pipedream/types";
+import certifier from "../../certifier.app.mjs";
+
+export default defineAction({
+  name: "Issue Credential",
+  version: "0.0.1",
+  key: "certifier-issue-credential",
+  description: "Create, issue and send a credential to a recipient.",
```

Comment:
```suggestion
  description: "Create, issue and send a credential to a recipient. [See the documentation](https://developers.certifier.io/reference/issue-a-credential)",
```

Reply:
This is probably not the best page we could link to. I see how the naming could be confusing 😞 I'll change the link to https://api.certifier.io/v1/credentials/create-issue-send, which is a similar multi-action.